src/MizarWidgets/CMakeLists.txt

```diff
@@ -35,11 +36,13 @@ add_executable(MizarWidgetsTests)
 target_sources(MizarWidgetsTests PRIVATE
   SamplingWithFrameTrackInputWidgetTest.cpp
   SamplingWithFrameTrackWidgetTest.cpp
+  SamplingWithFrameTrackReportConfigValidatorTest.cpp
 )
 
 target_link_libraries(MizarWidgetsTests PRIVATE
-  MizarWidgets
+  CONAN_PKG::abseil
   GTest::QtGuiMain
+  MizarWidgets
   Qt5::Test
   TestUtils
```

Comment:
In other cmakelists files we ordered this by: Orbit libraries first, then the others. (I don't actually care, just fyi)

Reply:
Thank you, I've never noticed that.